casanovo/denovo/model.py

```diff
@@ -139,6 +139,8 @@ def __init__(
             dropout=dropout,
             residues=residues,
             max_charge=max_charge,
+            enzyme_size=9,
+            padding_idx=9,
```

Comment:
Yes. The padding index is arbitrary but I felt it was best to have it as the "last" index of the embedding matrix and so because of the zero-indexing it could just as well be the same as the enzyme_size.